src/flake8/options/aggregator.py

```diff
@@ -8,9 +8,11 @@
 import argparse
 import configparser
 import logging
+import re
 from typing import Sequence
 
 from flake8.options import config
+from flake8.options.config import _find_config_file
```

Comment:
private names should not be imported (also it's possible the values below don't come from those config files but instead from plugins -- so I don't think it's right to show the filename anyway)

Reply:
good point, listing the invalid error code in the warning should already give the user enough information to know where it's coming from